appveyor.yml

```diff
@@ -12,12 +12,12 @@ branches:
 install:
   # 1. Select the right node
   # The following command works by fully reinstalling a version of node. It's a
-  # lot slower than the other command. This needs the full version (eg: 16.12)
-  # - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:platform
+  # lot slower than the other command. This needs the full version (eg: 18.12)
+  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:platform
   # Use the following command if the wanted version is already part of
   # appveyor's windows machines, see https://www.appveyor.com/docs/windows-images-software/#node-js
-  # This needs the major version only (eg: 16)
-  - ps: Install-Product node $env:nodejs_version $env:platform
+  # This needs the major version only (eg: 18)
+  # - ps: Install-Product node $env:nodejs_version $env:platform
```

Comment:
I wonder how much slower it will be compared to the older one. Maybe we should think about using circleci for the windows builders indeed.

Reply:
It's not clear to me whether circleci would be faster :)